tests/utils.py

```diff
@@ -73,16 +73,19 @@ def assert_span_http_status_code(span, code):
 
 
 @contextlib.contextmanager
-def override_env(env):
+def override_env(env, replace_os_env=False):
     """
     Temporarily override ``os.environ`` with provided values::
 
         >>> with self.override_env(dict(DD_TRACE_DEBUG=True)):
             # Your test
     """
-    # Copy the full original environment
+    # Only copy the full original environment if requested:
```

Comment:
By `if requested` we just mean if this util method is called?

Reply:
I was implying "if `replace_os_env` is False", but I can reword.